xml/bura-freezer_agent.xml

```diff
@@ -78,6 +85,12 @@
       </para>
      </listitem>
     </itemizedlist>
+    <note>
+     <para>
+      It is recommended that you use snapshots if the mode is MySQL, SQL
+      Server, or Mongo.
```

Comment:
Is MongoDB still supported? (I suspect I need to do MySQL->MariaDB here anyway.)